webapp/.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*.{js,jsx,ts,tsx,vue}]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Do we really need this file? I would assume IDE will pick up the prettier config instead of this one.

Reply:
I think it's worthwhile.